examples/air-purifier-app/linux/README.md

```diff
@@ -0,0 +1,109 @@
+# CHIP Linux Air Purifier Example
+
+An example showing the use of CHIP on the Linux. The document will describe how
+to build and run CHIP Air Purifier Example on A Linux System. This doc is tested
+on **Ubuntu 20.04 LTS**.
+
+The Air Purifier is a composed device with Endpoint 1 being the Air Purifier.
+Endpoint 2 is an Air Quality Sensor, Endpoint 3 is a Relative Humidity Sensor
+and endpoint 4 is a Temperature Sensor.
+
+To cross-compile this example on x64 host and run on **NXP i.MX 8M Mini**
+**EVK**, see the associated
+[README document](../../../docs/guides/nxp_imx8m_linux_examples.md) for details.
+
+<hr>
+
+-   [CHIP Linux Air Purifier Example](#chip-linux-air-purifier-example)
+    -   [Building](#building)
+    -   [Commandline arguments](#commandline-arguments)
+    -   [Running the Complete Example on Raspberry Pi 4](#running-the-complete-example-on-raspberry-pi-4)
+
+<hr>
+
+## Building
+
+-   Install tool chain
+
+          $ sudo apt-get install git gcc g++ python pkg-config libssl-dev libdbus-1-dev libglib2.0-dev ninja-build python3-venv python3-dev unzip
+
+-   Build the example application:
+
+          $ cd ~/connectedhomeip/examples/air-purifier-app/linux/
```

Comment:
please also add build_examples.py targets and implementation.

It seems you need to update:
  - scripts/build/build/targets.py
  - scripts/build/builders/ameba.py and cc32xx.py and host.py (for linux builds)
  
We may also want to pick one of these platforms to be included i CI to at least have a pass over the common bits.

Reply:
Hi @andy31415 - I've added the example app to Linux CI

![image](https://github.com/project-chip/connectedhomeip/assets/6282802/be6afddb-dd3c-4dfe-b36d-e7e26c75b518)

I will follow up with separate PR's for Ameba and TI as I need to get Bosch to test the TI version and we need to get a PR merged on the Ameba side with a change to their build container in order to build the new example with the scripts.

Reply:
Raised #29938 and #29939